content/en/agent/guide/python-3.md

```diff
@@ -9,42 +13,16 @@ This guide provides information and best practices on migrating checks between P
 
 To provide flexibility in allowing code to run multiple on versions of the Agent, this guide focuses on retaining backwards compatibility.
 
```

Comment:
To tie things together, we should keep a link to the `Use Python 3 with Datadog Agent v6` page, and to the `Upgrade to Datadog Agent v7` page.

We should mention in this page that the regular process to migrate to python 3 would be:

1. follow instructions below to make custom checks work with both Python 2 and 3. They can also use the custom check compatibility page: https://app.datadoghq.com/compatibility_check?config_compatibility_check=true
2. test switching the v6 Agent to Python 3 when all custom checks seem to support Python 3
3. once all custom checks are confirmed support Python 3, switch to v7 Agent

cc @jimmycaputo : let's discuss today how it'd make the most sense to document this "migration path"

Reply:
We could add this process to the "upgrade to Agent v7" page.. Waiting your feedback to see the final upgrade process.

Reply:
Yes I think we should add this to the "Upgrade to Agent V7" page at the top. We don't want customers who have custom checks to install v7 until they've gone through this process. You have it on the "Agent Version Differences" page, but someone might go directly to "Upgrade to Agent v7" and miss that. At the top of "Python 3 Custom Check Migration" we should mention that once the custom checks are migrated, they can / should update to Agent v7.